accord-core/src/main/java/accord/topology/Topologies.java

```diff
@@ -298,10 +308,18 @@ public boolean contains(Id to)
             return false;
         }
 
+        @Override
+        public int estimateUniqueNodes()
```

Comment:
This seems like a hefty over estimate if it is used allocate memory. Can't we precompute the precise answer and then memoize it and maybe combine with constructing the collection? Seems like in most cases we end up calling `Topologies.nodes()` and that calculates the precise set of things anyways so maybe just do it in advance?

Reply:
Is it a hefty calculation? It's _min_, so at e.g. rf=9 and 2 concurrent topologies it will allocate 10.

I don't mind memoizing, but this is not likely to be invoked more than once or so per lifetime of a `Topologies` and we'll probably allocate more memory calculating an accurate figure than we will guessing incorrectly.

Reply:
Yeah seems fine. I think I thought `size()` was a deep size not number of topologies.